modules/core/site.css

```diff
@@ -329,4 +329,105 @@ div.unseen, .unseen .subject { font-weight: 700; }
 }
 .drag_target {
     background-color: #888 !important;
+}
+
+.modal {
```

Comment:
All this will need to move to Bootstrap 5 soon: https://github.com/cypht-org/cypht/pull/828